code/game/machinery/doors/spacepod.dm

```diff
@@ -0,0 +1,24 @@
+/obj/structure/spacepoddoor
```

Comment:
You sure you can't make it /obj/machinery/door?

Reply:
it functions differently from a door